internal/tiltfile/os/os.go

```diff
@@ -164,6 +168,17 @@ func abspath(t *starlark.Thread, path string) (starlark.Value, error) {
 	return starlark.String(starkit.AbsPath(t, path)), nil
 }
 
+func relpath(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	var basepath, targpath string
+	if err := starkit.UnpackArgs(thread, fn.Name(), args, kwargs,
+		"basepath", &basepath,
```

Comment:
might wanna swap the arg order to allow compatibility with python

https://docs.python.org/3/library/os.path.html#os.path.relpath

Reply:
and it looks like defaulting the basepath to cwd would even be good for the docs!

Reply:
oh ugh you're right -- i did it this way to mirror the Go func but this makes much more sense 👍